spec/views/devise/sessions/new.html.slim_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     expect(rendered).
       to have_link(
-        t('links.create_account'), href: sign_up_email_path
+        t('links.create_account'), href: sign_up_email_url(request_id: nil)
```

Comment:
my memory is that `_url` came up only because the HTTP spec requires it for Redirect responses. That's all. I agree that `_path` is fine for href values etc, since those are all client-side anyway.